app/lib/publisher/models.dart

```diff
@@ -99,4 +99,16 @@ class PublisherMember extends db.ExpandoModel {
   /// One of [PublisherMemberRole].
   @db.StringProperty(required: true)
   String role;
+
+  /// Returns a new [PublisherMember] object with a new parent.
+  /// Should be used only for merging users.
+  PublisherMember changeParentUserId(String userId) {
```

Comment:
Can't we make this a utility method in `user_merger.dart`. Once we're done with `user_merger.dart` it might be better to delete it, rather than maintain it?